gateways/orbital/types.go

```diff
@@ -0,0 +1,150 @@
+package orbital
+
+import "encoding/xml"
+
+type RequestType string
+
+const (
+	RequestTypeNewOrder = "NewOrder"
+	RequestTypeCapture  = "MarkForCapture"
+	RequestTypeVoid     = "Reversal"
+)
+
+type MessageType string
+
+const (
+	MessageTypeAuth           MessageType = "A"
+	MessageTypeAuthAndCapture MessageType = "AC"
+	MessageTypeCapture        MessageType = "FC"
```

Comment:
Can we call this `MessageTypeForceAndCapture`? It seems like there is a slight difference between a Force and Capture and a normal capture, and I want to make sure they don't get confused here

Reply:
Sure thing, just to be clear though, the Capture method i implemented actually uses the "MarkForCapture" request type, as opposed to a NewOrder request with the AuthAndCApture or ForceCapture message types. 
Similar to reversal vs refund earlier, is it that the method of capturing we're interested in should use the NewOrder request and not MarkForCapture ?

Reply:
We want to use the "MarkForCapture" request type for the Capture method like you've already implemented. I just wanted the naming of this message type to make it clear that it is not a secondary method of performing a transaction and actually does something different.